omaha/main.scons

```diff
@@ -158,6 +158,7 @@ win_env = Environment(
 win_env['ENV']['GOROOT'] = os.getenv('GOROOT',
                                      win_env.subst('$MAIN_DIR') +
                                      '\\..\\third_party\\go\\files')
+win_env['ENV']['LocalAppData'] = os.getenv('LocalAppData')
```

Comment:
Not a big deal, why are we pulling in the LocalAppData into the environment?

Reply:
With Go1.12 build cache is required, so the environment should define GOCACHE or LocalAppData.